_1_basics/src/main/java/code/_1_study/_3_summing_non_floating/SummingNonFloatingTypes.java

```diff
@@ -12,19 +12,22 @@ private static void sumBytes() {
     byte y = 10;
     byte sum;
     sum = (byte) (x + y);
+    System.out.println(sum);
```

Comment:
am rugamintea sa nu faci modificari decat in folderele code._3_in_class si code._4_student_effort